backend/src/server.ts

```diff
@@ -77,23 +77,48 @@ const initializeServer = async (): Promise<void> => {
   server.log.info('registered sequelizePlugin...')
 
   const arkaConfigRepository = new ArkaConfigRepository(server.sequelize);
-  const configDatas = await arkaConfigRepository.findAll();
-  const configData: ArkaConfig | null = configDatas.length > 0 ? configDatas[0] : null;
 
   await server.register(fastifyCron, {
     jobs: [
       {
         // Only these two properties are required,
         // the rest is from the node-cron API:
         // https://github.com/kelektiv/node-cron#api
-        cronTime: configData?.cronTime ?? '0 0 * * *', // Default: Everyday at midnight UTC,
+        cronTime: process.env.PRICE_UPDATE_CRON_EXP ?? '0 0 * * *', // Default: Everyday at midnight UTC,
         name: 'PriceUpdate',
 
         // Note: the callbacks (onTick & onComplete) take the server
         // as an argument, as opposed to nothing in the node-cron API:
         onTick: async () => {
+          let configData: any
           if (process.env.CRON_PRIVATE_KEY) {
-            const paymastersAdrbase64 = configData?.deployedErc20Paymasters ?? ''
+            const unsafeMode = process.env.UNSAFE_MODE === "true" ? true : false;
+            if(!unsafeMode) {
+              const client = new SecretsManagerClient();
+              const api_key = process.env.DEFAULT_API_KEY;
+              const prefixSecretId = "arka_";
+              const AWSresponse = await client.send(
+                new GetSecretValueCommand({
+                  SecretId: prefixSecretId + api_key,
+                })
+              );
+              const secrets = JSON.parse(AWSresponse.SecretString ?? '{}');
+              configData = {
+                coingeckoApiUrl: secrets["COINGECKO_API_URL"],
```

Comment:
i need someone to verify if the keys which i have used are valid on secrets manager:
COINGECKO_API_URL, COINGECKO_IDS, CUSTOM_CHAINLINK_DEPLOYED, DEPLOYED_ERC20_PAYMASTERS, PYTH_MAINNET_CHAINIDS, PYTH_MAINNET_URL, PYTH_TESTNET_CHAINIDS, PYTH_TESTNET_URL